language/compiler/ir-to-bytecode/syntax/src/lexer.rs

```diff
@@ -332,14 +351,24 @@ fn get_name_len(text: &str) -> usize {
         .unwrap_or_else(|| text.len())
 }
 
-// Return the length of the substring containing characters in [0-9].
-fn get_decimal_digits_len(text: &str) -> usize {
-    text.chars()
+fn get_decimal_number(text: &str) -> (Tok, usize) {
```

Comment:
1) Precisely. Current plan is to prioritize on adding the two new types, so this easy-to-implement approach was chosen for the syntax. We can revise it later if we wish.
2) Parsing. It will fail when we we call `"257".parse::<u8>()`.